client/eip712.go

```diff
@@ -0,0 +1,55 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/evmos/ethermint/ethereum/eip712"
+)
+
+const FlagFeeDelegation = "--fee-delegation"
+
+func GenEIP712DataTypeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "gen-eip712-data-type <msgurl>",
```

Comment:
there should be a `eip712`cmd under `<app_name>d tx` so that you later add more subcommands. For example

```
evmosd tx eip712 data-type <msg-url>
```

Reply:
I think the `tx` subcommand is for building txs, I made an eip712 sub command in the root.